tests/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# NOTE: We're not using CTest here as just adds an extra layer around QuickTest
```

Comment:
but CTest adds a make test target that runs all tests, that is helpful for the CI